packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -242,6 +247,32 @@ export async function runBuildAndSubmitAsync(
   const haveAllBuildsFailedOrCanceled = builds.every(
     build => build?.status && [BuildStatus.Errored, BuildStatus.Canceled].includes(build?.status)
   );
+
+  const simBuilds = builds.filter(
```

Comment:
```suggestion
  const simBuilds = builds.filter(truthy).filter(
```

so you don't need `build?` everywhere (+ `assert`)